spec/models/manageiq/providers/ovirt/infra_manager/vm/reconfigure_spec.rb

```diff
@@ -421,6 +421,14 @@
           expect(vlans).to match_array([lan_mgmt.name, lan_A.name, "#{ext_lan.name}/#{ext_dist_switch.name}"])
         end
 
+        context 'with a powered off vm' do
+          let(:vm) { FactoryBot.create(:vm_ovirt, :ext_management_system => ems, :storage => storage, :raw_power_state => 'down') }
+
+          it 'returns only external lans' do
```

Comment:
I want to test the `parent_cluster` path but will need to set up the whole ems_metadata relationships tree for that to work

Reply:
Forgot how easy it was with the `add_child` method, done